tests/test_access_control.py

```diff
@@ -60,9 +60,10 @@ def test_ldap_host_base_not_defined(self, client, session):
         r = client.post('/api/v1.0/waivers/', data=json.dumps(self.data),
                         content_type='application/json', headers=self.headers)
         res_data = json.loads(r.get_data(as_text=True))
-        assert r.status_code == 500, r.text
-        assert res_data['message'] == ('LDAP_HOST and LDAP_SEARCHES also need to be defined '
-                                       'if PERMISSIONS is defined.')
+        assert r.status_code == 403, r.text
```

Comment:
Why was the status code changed? This should be HTTP 500 if the LDAP is not configured.

Reply:
This change came from your keycloak commit: [https://github.com/release-engineering/waiverdb/commit/36eab404e51ae8a50c71035e6d207d97c1b833b8#diff-5aa5097124803ba0c7e6b089af6daa600d6939df06461099637bdfb6aca5b79a](here)
I think, that `verify_authorization` method should work even when there's no way to get group information, cause we still can check permissions for the username.

Reply:
Ah, OK, I guess my thought process was: If the LDAP is not configured, just skip it and do authentication just based on the username. Let's keep it then.